packages/plugins/src/api/index.js

```diff
@@ -18,13 +18,13 @@ import { isFunction } from 'lodash';
  *
  * @property {string}                    name   A string identifying the plugin. Must be
  *                                              unique across all registered plugins.
- *                                              unique across all registered plugins.
  * @property {string|WPElement|Function} icon   An icon to be shown in the UI. It can
  *                                              be a slug of the Dashicon, or an element
  *                                              (or function returning an element) if you
  *                                              choose to render your own SVG.
  * @property {Function}                  render A component containing the UI elements
  *                                              to be rendered.
+ * @property {string}                    area   The pluginArea name to be rendered inside of.
```

Comment:
One more thing that I'm still not sure about. Should this scope default to `default` string when not provided? It might help to explain the idea - there is always some scope, right?